Header header

float32 torso_orientation
int16 torso_servo_speed

float32[6] head_servo_angles
int16[6] head_servo_speeds
string head_pose_frame_id
geometry_msgs/Pose head_pose
bool is_head_pose_reachable
